_posts/2018-10-25-released-v5.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Released Chainer/CuPy v5.0.0
+layout: post
+categories: Announcement
+author: Seiya Tokui
+---
+
+We have released Chainer and CuPy v5.0.0 today!
+This is a major release that introduces several new features.
+
+The following is a list of selected updates. Full updates can be found in the release notes: [Chainer](https://github.com/chainer/chainer/releases/tags/v5.0.0), [CuPy](https://github.com/cupy/cupy/releases/tag/v5.0.0).
+
+- **Static subgraph optimization** *(experimental)*.
```

Comment:
```suggestion
- **Static subgraph optimization** *(experimental)*. (Code name: Uneager Execution)
```

Reply:
Isn't it confusing? We are not using such a code name and also this feature is not about eagerness as I understand (the first iteration anyway runs in define-by-run style).